packages/ui/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @soramitsu-ui/ui
 
+## 0.10.2
+
+### Patch Changes
+
+- 6095b73: **feat**(`STable`): now `empty` slot have no wrappers with styles and added new `empty-text` slot, that works like old `empty` slot
```

Comment:
New slot is a minor change, not patch.

Reply:
It's so small I thought patch whould be ok. I'll change to minor then